spec/support/features/in_person_helper.rb

```diff
@@ -29,6 +29,17 @@ def fill_out_state_id_form_ok
     fill_in t('in_person_proofing.form.state_id.state_id_number'), with: GOOD_STATE_ID_NUMBER
   end
 
+  def fill_out_state_id_form_ok_include_address
+    fill_out_state_id_form_ok
+    fill_in t('in_person_proofing.form.state_id.address1'), with: GOOD_ADDRESS1
+    fill_in t('in_person_proofing.form.state_id.address2'), with: GOOD_ADDRESS2
+    fill_in t('in_person_proofing.form.state_id.city'), with: GOOD_CITY
+    select GOOD_STATE_ID_JURISDICTION,
+           from: t('in_person_proofing.form.state_id.state_id_jurisdiction')
+    fill_in t('in_person_proofing.form.state_id.zipcode'), with: GOOD_ZIPCODE
+    choose t('in_person_proofing.form.state_id.id_address_current_no')
```

Comment:
The spec fails on this line with the below error
```
Capybara::ElementNotFound:
       Unable to find visible radio button "No, my current residential address is not listed on my state-issued ID" that is not disabled
```
When I ran the test with the browser showing, the radio buttons displayed.  I'm not sure why this is failing.

Reply:
i wonder if part of the issue is that the above test likely needs to be changed as well considering the order of the form elements has changed. so `fill_out_state_id_form_ok` currently has the order:
1. first name
2. last name
3. dob: month, day, year
4. state_id_jurisdiction
5. state_id_num
Then we're filing in:
6. address1
7. address2
8. city
9. state_id_jurisdiction
10. zip
Then we expect radio buttons

But the new order of the form is 
1. first name
2. last name
3. dob: month, day, year
4. state_id_num
5. address1
6. address2
7. city
8. state_id_jurisdiction
9. zip
Then we expect radio buttons

Reply:
ok wait the layout only changes if `in_person_capture_secondary_id_enabled` is true right? So then instead of changing the test before this one I wouldn't include it on line 33 and would instead fill the form out with the new order

Reply:
I don't think it has to do with the order.  This test failed before I added `fill_out_state_id_form_ok` to this test.  Here's [the commit where I made the change](https://github.com/18F/identity-idp/pull/7913/commits/3ae6c3a38ab402a777a086927c78c8a2aa2afb30).  Also, if you look at this video of the failing feature test, there don't seem to be any problems with filling out the form before the radio buttons.

https://user-images.githubusercontent.com/80347702/222504179-d4995f4e-faa5-4040-b61a-3d5a31566873.mov

Reply:
i see what you're saying